<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>384</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="save_Button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_Button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>교회명</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="church_name_widget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>128</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>주  소</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="church_addr_widget">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>130</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>고유번호</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="church_biz_widget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>로고등록</string>
   </property>
  </widget>
  <widget class="QPushButton" name="file_select_Button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>파일선택</string>
   </property>
  </widget>
  <widget class="QLabel" name="selected_logo_file_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="denomination_name_widget">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>41</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>41</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>교단명</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close_exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>church_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>file_select_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>select_logo_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>church_save()</slot>
  <slot>close_exit()</slot>
  <slot>select_logo_file()</slot>
 </slots>
</ui>
